### DISCUSSION TITLE
留言与建议

### DISCUSSION THREAD

**Discussion by shenweiyan** [shenweiyan/Digital-Garden#16] [x.x-留言]
如果你通过本人的博文/平台学到了一点东西，那么对我最好的奖赏就是请你也尝试教我些东西。你可以通过邮箱、微信，或者移步到 [GitHub Discussions](https://github.com/shenweiyan/Digital-Garden/discussions/16) 写下您的建议，或者向我提问。

感谢你的一路支持！

📢 如果给我发邮件，或者通过微信添加好友，请写上您的**真名实姓**，让我感受到一个**真实的人的气息**。我不太愿意跟**网名**打交道，对于那些不知来路、上来就问问题的微信和邮件，我通常会直接忽略。

- 微信：**ishenweiyan**（请备注真实姓名，谢谢）
- 邮箱：**shenweiyan@aliyun.com**

<p align="center">
    <img src="https://gi.weiyan.tech/0000/img/feedback.gif" alt="feedback">
</p>

**Comment by shenweiyan** [comment]
想了一下，还是增加一个留言的页面吧 >.<

**Reply by QiYongchuan** [reply]
很酷的博客！无论是页面风格还是内容真厉害，向大佬学习！

**Comment by vbskycn** [comment]
hello

**Comment by daoyuc** [comment]
我也去整个webstack导航~~ 一入hugo深似海，从此wp是路人。

**Reply by shenweiyan** [reply]
搞起搞起 😎

**Comment by izshui** [comment]
天气插件应该如何做呢？是在headers.yml文件中添加吗？

**Reply by shenweiyan** [reply]
在 `content_header.html` 文件中修改，和风天气的[天气插件](https://widget.qweather.com/)在2024年5月1日已经停止服务了，建议可以使用心知天气。

我们在 [NavStack-Hugo-Theme](https://github.com/PageXHub) 中做了尝试，暂时未开发给大家使用。

**Comment by hw2001aq** [comment]
感谢博主分享导航源码，真的不错！
现在考虑怎么整合导航和博客？有教程指导吗？

**Reply by shenweiyan** [reply]
目前导航和博客是分开的，Hugo 的博客主题非常多，可以随便选，但是在现在的导航中再增加博客，这一点我还没想好，暂时没有教程。

**Comment by Wcowin** [comment]
非常好的博客！学习了！

**Comment by hw2001aq** [comment]
请问博主，导航部署后电脑浏览器可以打开，手机上打不开，不知道什么原因？

**Reply by shenweiyan** [reply]
> 请问博主，导航部署后电脑浏览器可以打开，手机上打不开，不知道什么原因？

你的站点是部署在哪里？通过 IP 还是域名访问？信息有点少，暂时判断不出哪里出问题了。另外，导航的问题，可以到导航对应的仓库提交 Issues 进行讨论，谢谢。

> 是MKdocs吗？你描述详细一点，或许i can help you

导航站点应该指的是 [WebStack-Hugo](https://github.com/shenweiyan/WebStack-Hugo)，这是基于 Hugo 的。

**Comment by jaywhj** [comment]
过来学习了🤣
可以考虑给其它页面也开通留言功能，这样更方便交流😎

**Reply by shenweiyan** [reply]
其他页面的底部都接入了 giscus，应该都是有留言功能的 >.<
